app/content-field-type-options/number/controller.js

```diff
@@ -1,2 +1,17 @@
-﻿app.controller('contentFieldTypeOptionsNumberController', [function () {
+﻿app.controller('contentFieldTypeOptionsNumberController', ['$scope', function ($scope) {
+
+    // set default options
+    if (!$scope.contentField.options.controlType) {
+        $scope.contentField.options.controlType = 'textbox';
+    }
+    if (!$scope.contentField.options.step) {
+        $scope.contentField.options.step = 1;
+    }
+    if (!$scope.contentField.options.minValue) {
+        $scope.contentField.options.minValue = 0;
+    }
+    if (!$scope.contentField.options.maxValue) {
+        $scope.contentField.options.maxValue = 100;
```

Comment:
Do we want to enforce those values if not specified or use the min-max from the data-type (I assume int32 ?)
If my assumption is right, it would be:

- Min: -2147483648
- Max: 2147483647

Reply:
I wasn't thinking about the allowable DB values at this point.  This code was just for setting the min/max for the range control and the options in the dropdown control.  

The editor won't know what type the actual database field is.

Reply:
@valadas I think these are just the `default values` for min/max when not present otherwise.